### PR TITLE
#77 Added refreshing access token

### PR DESCRIPTION
**Pull request by Chubatik**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -22,6 +24,13 @@ export const AppRouting = () => (
 )
 
 const App = () => {
+  // Refresh access token every 15 minutes
+  useEffect(() => {
+    setInterval(async () => {
+      const token = await api.get(`${API_URL}/token`)
+      await Cookies.set('accessToken', token.headers['accesstoken'])
+    }, REFRESH_TOKEN_DURATION)
+  }, [])
```
Comment (irenlian) [inline src/App.tsx:33]: If you'd like to have asynchronous action, I'd recommend you to consider web-worker

Comment (Chubatik) [reply src/App.tsx:33]: So i need to call api.get() inside worker, right?

Comment (Chubatik) [reply src/App.tsx:33]: Or if remove async from setTimeout, could I avoid using worker ?

Comment (irenlian) [reply src/App.tsx:33]: You can't avoid async and this won't impact the fact that a server call happens in the main thread.
This is just my recommendation.

**Review by KolomiietsAn — APPROVED** [review]

**Pull request merged**